src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- SYSTEM_NET_PRIMITIVES_DLL is required to allow source-level code sharing for types defined within the
          System.Net.Internals namespace. -->
     <DefineConstants>$(DefineConstants);SYSTEM_NET_PRIMITIVES_DLL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
It looks like there is a uap but not a uapaot configuration https://github.com/tijoytom/corefx/blob/4e9a2dd415434c3f40b55564d2a29be5e3547554/src/System.Net.Primitives/src/Configurations.props. If you want this to be uapaot specific then there will need to be a new configuration added.